tests/ban_test.py

```diff
@@ -0,0 +1,32 @@
+import os
+import logging
+import unittest
+from unittest.mock import Mock, MagicMock, PropertyMock, patch
+
+from errbot.backends.test import TestBot
+
+import plugins.ban
+
+from tests.helper import plugin_testbot
+
+
+class TestBan(unittest.TestCase):
+
+    @patch('plugins.ban.requests')
+    @patch('plugins.ban.json')
+    def test_ban_cmd(self, mockjson, mockreq):
+        ban, testbot = plugin_testbot(plugins.ban.Ban, logging.ERROR)
+        ban.activate()
+
+        ban.bot_config.ROOMS_TO_JOIN = ('coala/coala', 'coala/coala-bears')
+        ban.bot_config.BOT_IDENTITY['token'] = 'mocked?'
+
+        fake_room_data = [{'id': '130', 'uri': 'coala/coala'},
+        		  {'id': '234', 'name': 'Nitanshu'},
```

Comment:
horrific style.
please get pyflakes active on these files.

Reply:
Sorry, that was looking regularly indented in vim.